src/components/ZetkinDateTime.stories.tsx

```diff
@@ -0,0 +1,26 @@
+import { ComponentMeta, ComponentStory } from '@storybook/react';
+
+import ZetkinDateTime from './ZetkinDateTime';
+
+export default {
+  component: ZetkinDateTime,
+  title: 'Atoms/ZetkinDateTime',
+} as ComponentMeta<typeof ZetkinDateTime>;
+
+const Template: ComponentStory<typeof ZetkinDateTime> = (args) => (
+  <ZetkinDateTime
+    convertToLocal={args.convertToLocal}
+    datetime={args.datetime}
+  />
+);
+
+export const basic = Template.bind({});
+basic.args = {
+  datetime: new Date().toISOString().slice(0, 19),
+};
+
+export const convertToLocal = Template.bind({});
+convertToLocal.args = {
+  convertToLocal: true,
+  datetime: new Date().toISOString().slice(0, 19),
```

Comment:
I feel like there should be a built in method for this, or you can use `.toJson()`

Reply:
from my research now, the `toJSON()` still puts a Z at the end. The option people seem to do is either slicing out the Z, or doing `new Date().format('YYYY-MM-DD[T]HH:mm:ss')`. Which approach do you think makes more sense? I think either way I should add a comment of why the slicing or formatting is happening.

Reply:
Oh really `toJson` puts a Z at the end woah. Ok, hm I say then stick with the slice and as you said, use a comment :)